workflowactions.md

```diff
@@ -0,0 +1,31 @@
+# Workflow Actions Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+A workflow action represents a single action in the reviewing process, e.g. the accept/reject action.  
+
+All endpoints mentioned here require authentication, but no specific permissions.
+
+## Main Endpoint
+**/api/config/workflowactions**   
```

Comment:
It's a little odd to me that this is not under `/api/workflow/workflowactions`.  Is there a reason for putting this under `/config`?

Reply:
This endpoint only exposes the configured potential actions similar to https://github.com/DSpace/Rest7Contract/blob/master/workflowdefinitions.md and https://github.com/DSpace/Rest7Contract/blob/master/workflowsteps.md

I've used the config category to remain in line with the contracts related to https://github.com/DSpace/Rest7Contract/blob/master/submissionforms.md

Reply:
Thanks for the explanation! I obviously reviewed this too quickly as your choice now seems obvious to me.  I'll close this question as resolved.